src/supermarket/app/models/quality_metric.rb

```diff
@@ -25,4 +25,8 @@ def self.license_metric
   def self.supported_platforms_metric
     QualityMetric.where(name: 'Supported Platforms').first_or_create!(admin_only: true)
   end
+
+  def self.contributing_file_metric
+    QualityMetric.find_by(name: 'Contributing File')
```

Comment:
What are your thoughts around this finder not following the `QM.where(name:).first_or_create!()` pattern of the others?

In the last round of metric work, I changed these so that both the seeds and the class finder methods would ensure these fixture-y records existed. I'm open to dialing the creation back to only seeds, but I suggest these finder methods should be consistent with each other.